pushy-benchmark/src/main/java/com/relayrides/pushy/apns/util/ApnsPayloadBuilderBenchmark.java

```diff
@@ -0,0 +1,75 @@
+package com.relayrides.pushy.apns.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+
+@State(Scope.Thread)
+public class ApnsPayloadBuilderBenchmark {
+
+    private ApnsPayloadBuilder apnsPayloadBuilder;
+
+    private static final int MAXIMUM_PAYLOAD_SIZE = 1024 * 8;
+
+    private String shortAsciiMessageBody;
+    private String longAsciiMessageBody;
+    private String shortChineseMessageBody;
+    private String longChineseMessageBody;
+
+    @Setup
+    public void setUp() {
+        this.apnsPayloadBuilder = new ApnsPayloadBuilder();
+
+        this.shortAsciiMessageBody = RandomStringUtils.randomAscii(MAXIMUM_PAYLOAD_SIZE / 8);
```

Comment:
do you see any differences between re-runs of the test harness?  It might be good to mutate these input messages somehow between iterations, if so, to "smooth out" any effects of the random input data.